recipes/mdahole2/build_base.sh

```diff
@@ -0,0 +1 @@
+${PYTHON} -m pip install . --no-deps -vv
```

Comment:
We need all those options to avoid a bug in multiple outputs.

```suggestion
${PYTHON} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
Thanks, added!